src/Symfony/Component/Messenger/Bridge/AmqpExt/composer.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "symfony/messenger-amqp",
```

Comment:
shouldn't it be `symfony/amqp-messenger` for consistency with `symfony/slack-notifier` and `symfony/mailchimp-mailer`?

Reply:
Thank you.